### ISSUE TITLE
Replace static analysis to linear in `cantilever.jl`

### ISSUE THREAD

**Issue by mvanzulli** [mvanzulli/ONSAS.jl#500]
The example is linear, so there is no need to run a non-linear analysis.

**Comment by mvanzulli** [comment]
Sth like this: 

```julia
"Return the problem solution"
function solve()
    s = structure()
    # -------------------------------
    # Structural Analysis
    # -------------------------------
    (; NSTEPS) = parameters()
    sa = LinearStaticAnalysis(s; NSTEPS)
    # -------------------------------
    # Numerical solution
    # -------------------------------
    sol = solve!(sa)
end;
```

should be enough ⭕

**Comment by mvanzulli** [comment]
Closed via #525
